src/html/shared/accordion.html

```diff
@@ -0,0 +1,53 @@
+<div class="o-layout">
+    <div class="o-layout__item  u-1/4@tablet">
+        <div class="c-accordion">
+            <label class="c-accordion__header">
+                Footer Section 1 {{ sprite("plus", "0 0 1 1", "c-icon c-icon--tiny c-icon--darkbg c-accordion__icon") }}
+            </label>
+            <ul class="o-list-bare  c-accordion__list isCollapsed">
+                <li class="o-list-bare__item c-accordion__panel_text">Meet Shopify Light</li>
+                <li class="o-list-bare__item c-accordion__panel_text">Features</li>
+                <li class="o-list-bare__item c-accordion__panel_text">Another Link</li>
+            </ul>
+        </div>
+    </div>
+
+    <div class="o-layout__item  u-1/4@tablet">
+        <div class="c-accordion">
+            <label class="c-accordion__header">
+                Footer Section 2 {{ sprite("plus", "0 0 1 1", "c-icon c-icon--tiny c-icon--darkbg c-accordion__icon") }}
+            </label>
+            <ul class="o-list-bare  c-accordion__list isCollapsed">
+                <li class="o-list-bare__item c-accordion__panel_text">Meet Shopify Light</li>
+                <li class="o-list-bare__item c-accordion__panel_text">Features</li>
+                <li class="o-list-bare__item c-accordion__panel_text">Another Link</li>
+            </ul>
+        </div>
+    </div>
+
+    <div class="o-layout__item  u-1/4@tablet">
+        <div class="c-accordion">
+            <label class="c-accordion__header">
+                Footer Section 3 {{ sprite("plus", "0 0 1 1", "c-icon c-icon--tiny c-icon--darkbg c-accordion__icon") }}
+            </label>
+            <ul class="o-list-bare  c-accordion__list isCollapsed">
+                <li class="o-list-bare__item c-accordion__panel_text">Meet Shopify Light</li>
+                <li class="o-list-bare__item c-accordion__panel_text">Features</li>
+                <li class="o-list-bare__item c-accordion__panel_text">Another Link</li>
+            </ul>
+        </div>
+    </div>
+
+    <div class="o-layout__item  u-1/4@tablet">
+        <div class="c-accordion">
+            <label class="c-accordion__header">
```

Comment:
Add `aria-controls` and `aria-controlled-by` ref: http://heydonworks.com/practical_aria_examples/